css/amp.css

```diff
@@ -144,31 +144,47 @@ html.i-amphtml-ios-embed.i-amphtml-ios-overscroll > #i-amphtml-wrapper {
   display: inline-block;
 }
 
-.i-amphtml-layout-nodisplay {
+.i-amphtml-layout-nodisplay,
+[layout=nodisplay]:not(.i-amphtml-layout-nodisplay)
+{
   /* Display is set/reset in JS */
 }
 
-.i-amphtml-layout-fixed {
+.i-amphtml-layout-fixed,
+/*, [width][height]:not([layout=fixed]) */
+[layout=fixed][width][height]:not(.i-amphtml-layout-fixed)
```

Comment:
This is a breaking change (almost in all the places), since we are increasing the specificity here. It is probably going to break a lot of pages.

Reply:
> Once it is resolved, the selector will fail to apply, so everything keeps the same specificity as before.